Cargo.toml

```diff
@@ -13,3 +13,10 @@ tokio = { version = "1.32.0", features = ["io-util"] }
 
 [dev-dependencies]
 tokio = { version = "1.32.0", features = ["macros", "net", "rt"] }
+
+[workspace]
+resolver = "2"
+members = [
+  ".",
```

Comment:
```suggestion
```

If I understand correctly, line 20 is not needed. See https://doc.rust-lang.org/cargo/reference/workspaces.html#root-package